aries-site/package.json

```diff
@@ -23,7 +23,7 @@
     "start": "next",
     "start-server": "yarn build && yarn next start -p 3030",
     "test": "yarn build && node src/scripts/start-tests.js",
-    "test:local": "start-server-and-test start-server 3030 'testcafe all src/tests/'",
+    "test:local": "start-server-and-test start-server 3030 'testcafe chrome,firefox src/tests/'",
```

Comment:
Another separate thought; Since we have a monorepo and I would think that for any core library change we want the pre-commit to run on the website as well, maybe it should mean the pre-commit command should be found on the parent package.json?

Reply:
@ShimiSun Take a look at `huskyrc.js` which sits at the monorepo root. It serves as the controller, which is currently configured to do what you are suggesting (i.e. run aries-site tests even when changes are made to aries-core).

Using `husky` package instead of `pre-commit` because I found it to easier to implement and understand in monorepo.